packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../utils/index';
+
+@mixin iui-expandable-block {
+  --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon);
+  @include iui-expandable-block-variant;
+  @include iui-expandable-block-size;
+
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-radius: var(--iui-border-radius-1);
+
+  &:where(.iui-expanded) {
+    --_iui-expandable-block-expander-icon-transform: rotate(90deg);
+  }
+}
+
+/// @arg $variant Can be one of: default, borderless
+@mixin iui-expandable-block-variant($variant: 'default') {
+  @if $variant == 'default' {
+    --_iui-expandable-block-border: 1px solid var(--iui-color-border);
+    --_iui-expandable-block-background-color: var(--iui-color-background);
+    --_iui-expandable-block-content-padding: var(--iui-size-s);
+    --_iui-expandable-block-header-background-color-disabled: var(--iui-color-background-disabled);
+    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-hover);
+    --_iui-expandable-block-header-focus-offset: -2px;
+    --_iui-expandable-block-header-focus-thickness: 2px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: 0;
+    }
+  }
+  @if $variant == 'borderless' {
+    --_iui-expandable-block-border: none;
+    --_iui-expandable-block-background-color: transparent;
+    --_iui-expandable-block-content-padding: none;
+    --_iui-expandable-block-header-background-color-disabled: transparent;
+    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-transparent-hover);
+    --_iui-expandable-block-header-focus-offset: -1px;
+    --_iui-expandable-block-header-focus-thickness: 1px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: inherit;
+    }
+  }
+}
+
+/// @arg $size Can be one of: default, small
+@mixin iui-expandable-block-size($size: 'default') {
+  @if $size == 'default' {
+    --_iui-expandable-block-expander-icon-size: var(--iui-size-m);
+    --_iui-expandable-block-header-gap: var(--iui-size-s);
+    --_iui-expandable-block-header-padding: var(--iui-size-s);
+  }
+  @if $size == 'small' {
+    --_iui-expandable-block-expander-icon-size: var(--iui-size-s);
+    --_iui-expandable-block-header-gap: var(--iui-size-xs);
+    --_iui-expandable-block-header-padding: 6px var(--iui-size-xs);
+  }
+}
+
+@mixin iui-expandable-block-header {
+  @include iui-reset;
+  @include iui-focus(
+    $offset: var(--_iui-expandable-block-header-focus-offset),
+    $thickness: var(--_iui-expandable-block-header-focus-thickness)
+  );
+  display: flex;
+  gap: var(--_iui-expandable-block-header-gap);
```

Comment:
Not sure, but I can't find `gap` on  property in the old code. Is this alright?

To confirm, is this PR trying to change the styling too apart from just adding the diabled state? If so, then maybe the missing 1-1 mappings of old-new code might be alright.

Reply:
You are correct, we were using `margin` on the nested classes instead of `gap`.  This was done because we were still supporting Internet Explorer and [IE does not support `gap`](https://caniuse.com/?search=gap).  Now we have dropped IE support so I'd like to use `gap` instead of the `margin` hack.

If you are ok with me keeping it I would prefer to, but if you'd rather we address that in a separate PR I am ok with that too.

Reply:
Sounds good, was just confirming, thanks. Since this PR is also improving the code structure, I think adding this change in this PR would be good.

Maybe we can also add that this PR is "improving the code structure" to the PR description.

Reply:
Description updated.